src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md

```diff
@@ -7,22 +7,24 @@ start_time: '10:00+02:00'
 end_date: '2024-03-20'
 end_time: '18:00+02:00'
 location_name: Ghent University
-address: TBD
+address: Hermann von Helmholtz, campus Ardoyen, iGent, 9052 Zwijnaarde, Belgium
 location_url: https://www.ugent.be/en
-# location_latlng: [] // TODO Fill this in once known
+location_latlng: [51.01287821684896, 3.708004390525348]
 ---
 
 Local event to be held at Ghent University, Belgium
 
+Registrations are open! Register [here](https://event.ugent.be/registration/event/138e28ec-3e42-4370-833d-628d578e047c) to secure your spot.
+
```

Comment:
We will have a centralized registration, where we will forward you the signups to Ghent. so no need for you to handle it
```suggestion
```

Reply:
Oh okay good to know, thanks :) when will this be available?

Reply:
we open the registration on february 2nd. you can ping @christopher-hakkaart if you need info at a certain date.

Reply:
Thank you! February 2nd is perfect :)